### ISSUE TITLE
No sound bug

### ISSUE THREAD

**Issue by Ariloum** [mdias/rs_asio#333]
Hello, looks like this version is not working on my side. I hear only short crackling sound on RS launch and then silence.

Old version (released in 2021 december) was working fine until RS updated and started showing the error "no RS cable connected, so it is still playing sound well but can't handle the inputs.

I tried to enable EnableRefCountHack = no luck, I tried custom/driver and different ASIO buffer sizes - nothing changed.

log is full of these errors:
0.111 [INFO]  {ASIO Out} xrun
0.111 [INFO]  ReaRoute ASIO - AsioSharedHost::AsioCalback_bufferSwitch - buffer switch 3 (not logging upcoming switches)
0.111 [INFO]  {ASIO Out} DebugWrapperRenderClient::GetBuffer NumFramesRequested: 256
0.111 [INFO]  {ASIO Out} recovered from 22 consecutive xruns. Ignoring for some time.
0.111 [INFO]  {ASIO Out} DebugWrapperRenderClient::ReleaseBuffer NumFramesWritten: 256
0.111 [INFO]  {ASIO Out} DebugWrapperRenderClient::GetBuffer NumFramesRequested: 256
0.111 [INFO]  {ASIO Out} DebugWrapperRenderClient::ReleaseBuffer NumFramesWritten: 256
0.111 [INFO]  {ASIO Out} RSAsioAudioClient::OnAsioBufferSwitch - failed to get lock first time. This might be harmless, but if we freeze here this is likely related
0.111 [INFO]  {ASIO Out
[errors.txt](https://github.com/mdias/rs_asio/files/10254902/errors.txt)
} RSAsioAudioClient::OnAsioBufferSwitch - failed to get lock first time. This might be harmless, but if we freeze here this is likely related
0.112 [INFO]  {ASIO Out} RSAsioAudioClient::OnAsioBufferSwitch - failed to get lock first time. This might be harmless, but if we freeze here this is likely related
0.112 [INFO]  {ASIO Out} recovered from 10 consecutive xruns. Ignoring for some time.

**Comment by mdias** [comment]
It seems you're trying to use Rearoute, which may have some implication.
Do you still have the problem if you use your interface directly instead of through Rearoute?

**Comment by Ariloum** [comment]
Looks like connecting directly to the audio interface's asio works just fine, so it's kind of rearoute related issue.

**Comment by mdias** [comment]
It could be that you have a too heavy processing chain going through ReaRoute and your machine just can't process it quickly enough.

**Comment by Ariloum** [comment]
I don't think so, previous version of rs-asio works just fine for audio output. also I got top notch ryzen 9 7950x cpu with 0 processing on the rea channels.

**Comment by Ariloum** [comment]
feel free to request any additional info or tests.. 
I have no idea why it is not working, but there is something different on latest version rs-asio compared to 2021 december version
